satellite/audit/service.go

```diff
@@ -30,6 +32,8 @@ type Config struct {
 	MinBytesPerSecond  memory.Size   `help:"the minimum acceptable bytes that storage nodes can transfer per second to the satellite" default:"128B"`
 	MinDownloadTimeout time.Duration `help:"the minimum duration for downloading a share from storage nodes before timing out" default:"25s"`
 	MaxReverifyCount   int           `help:"limit above which we consider an audit is failed" default:"3"`
+
+	Slots int `help:"number of reservoir slots allotted for nodes" default:"1"`
```

Comment:
Having this `Slots` config seems misleading if we use `2` for all values larger than `2`. I think we should specify that it will be capped at 2 in the help message.